qiskit_quantinuum/VERSION.txt

```diff
@@ -1 +1 @@
-0.3.0
\ No newline at end of file
+0.3.1
```

Comment:
I think we should probably use 0.4.0 since this is the first release with the new package name:

```suggestion
0.4.0
```